### ISSUE TITLE
Regarding the test set used in the Evaluation section of the paper

### ISSUE THREAD

**Issue by little-misfit** [pandayuanyu/generative-photography#4]
Great work! I have a question about the test set. The paper mentions that 75 camera setups were used for testing in each case. However, I couldn't find the corresponding test sample configurations in the dataset (which only includes the training and validation sets). Could you please provide them?
